go.mod

```diff
@@ -4,16 +4,25 @@ go 1.20
 
 require (
```

Comment:
It would be nice to upgrade vocdoni-node import, run `go get go.vocdoni.io/dvote@master` and adapt the code if there is some breaking change in the API. 

Thanks!